### ISSUE TITLE
Form labels have low contrast

### ISSUE THREAD

**Issue by FedeG** [eventoL/eventoL#800]
**Describe the bug**

The form labels have a font color with low contrast to the background

**To Reproduce**

1. Go to sign-up (or any form)
2. See the error

**Expected behavior**

The form labels should have a gray or black color with high contrast with respect to the background

**Screenshots**

![Screenshot from 2022-04-09 03-10-53](https://user-images.githubusercontent.com/4097554/162716408-e077e6e7-43f6-449f-a474-0f8d960205e7.png)

**Comment by FedeG** [comment]
# Fixed

![login_form](https://user-images.githubusercontent.com/4097554/162782479-689d89bf-ea03-4d4c-9314-639127c509b1.png)
![proposal_form](https://user-images.githubusercontent.com/4097554/162782482-fdde0e83-ddd7-49b8-8ef5-f0cd603d7ff9.png)
